commands/bridge_configure.go

```diff
@@ -7,12 +7,24 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/MichaelMure/git-bug/bridge/core"
+
 	"github.com/MichaelMure/git-bug/bridge"
 	"github.com/MichaelMure/git-bug/cache"
 	"github.com/MichaelMure/git-bug/util/interrupt"
 	"github.com/spf13/cobra"
 )
 
+const (
+	defaultName = "default"
+)
+
+var (
+	name         string
```

Comment:
those variables are in the same namespace as all commands, they need to be prefixed with the command's name

Reply:
Agreed ! changed the variables to `bridgeConfigureName` and `bridgeConfigureTarget`